jobs/config.json

```diff
@@ -11358,5 +11358,14 @@
     "sigOwners": [
       "sig-big-data"
     ]
+  },
+  "minigo-testing": {
```

Comment:
I feel like this needs to link into the config below somehow?

Reply:
is this yet another job?

Reply:
Ah, oops. This is me not understanding how this file works. Fixed now.